### PR TITLE
Support for future post status

### PR DESCRIPTION
**Pull request by 0xb4lint**

### CODE REVIEW

File: src/Model/Builder/PostBuilder.php [hunk 1]
```diff
@@ -26,7 +26,13 @@ public function status($status)
      */
     public function published()
     {
-        return $this->status('publish');
+        return $this->where(function ($query) {
+            $query->status('publish');
+            $query->orWhere(function ($query) {
+                $query->status('future');
+                $query->where('post_date', '<=', \Carbon\Carbon::now()->format('Y-m-d H:i:s'));
```
Comment (jgrossi) [inline src/Model/Builder/PostBuilder.php]: I think this should be `->whereDate()` and just `Carbon::now()`

**Review by jgrossi — CHANGES_REQUESTED** [review]
hey! thanks for the PR! good call. just a small change 💪🎉

File: src/Model/Builder/PostBuilder.php (hunk 1 above)
Comment (0xb4lint) [reply src/Model/Builder/PostBuilder.php]: I've changed `\Carbon\Carbon::now()` to `Carbon::now()`.

Please correct me if I'm wrong: as I can see using `whereDate()` generates a `strftime('%Y-%m-%d', "post_date") <= cast(? as text)` query (and using `whereTime()` generates `strftime('%H:%M:%S', "post_date") <= cast(? as text))` query).

So I think `where()` is the proper one in this case.

**Comment by 0xb4lint** [comment]
@jgrossi I've changed `\Carbon\Carbon::now()` to `Carbon::now()`.

Please correct me if I'm wrong: as I can see using `whereDate()` generates a `strftime('%Y-%m-%d', "post_date") <= cast(? as text)` query (and using `whereTime()` generates `strftime('%H:%M:%S', "post_date") <= cast(? as text))` query).

So I think `where()` is the proper one in this case.

File: src/Model/Builder/PostBuilder.php (hunk 1 above)
Comment (jgrossi) [reply src/Model/Builder/PostBuilder.php]: you're right! sounds good! thanks for sharing! 🎉🔥

**Comment by jgrossi** [comment]
@valentinxxx do you need this for `2.6` version? the current version (and branch) is `2.7`. any problem for you? thanks!

**Comment by 0xb4lint** [comment]
@jgrossi `2.7` version will be fine for me 😉 
Should I create a new PR?

**Review by jgrossi — APPROVED** [review]

**Comment by jgrossi** [comment]
@valentinxxx no worries. I already changed it to `2.7` and gonna merge now and release a new version in a couple of minutes. thanks!

**Pull request merged**